Makefile

```
@@ -390,6 +390,37 @@ lint-other-targets:
 	--all-features \
 	-- -D warnings
 
+lint-scroll-reth:
+	cargo +nightly clippy \
+	--workspace \
+	--bin "scroll-reth" \
+	--lib \
+	--examples \
+	--tests \
+	--benches \
+	--features "scroll $(BIN_OTHER_FEATURES) skip-state-root-validation" \
+	-- -D warnings
+
+lint-all:
+	cargo +nightly clippy \
+	--workspace \
+	--lib \
+	--examples \
+	--tests \
+	--benches \
+	--all-features \
+	--locked
+
+lint-udeps:
+	cargo udeps --workspace --lib --examples --tests --benches --all-features --locked \
```

I think we want to run this with nightly
```suggestion
	cargo +nightly udeps --workspace --lib --examples --tests --benches --all-features --locked \
```